client/src/modules/list/index.jsx

```diff
@@ -377,6 +377,10 @@ class List extends Component {
                   </button>
                   {isMembersBoxVisible && (
                     <MembersBox
+                      inviteData={{
+                        name,
+                        url: `sack/${listId}`
```

Comment:
you can use here our: `routeGenerator(Routes.LIST, listId)` helper from `helpers.js file`